requirements/posix.txt

```diff
@@ -1,3 +1,3 @@
 ipykernel
 cloudpickle
-wurlitzer
\ No newline at end of file
+wurlitzer>=1.0.3
```

Comment:
Please add this requirement to our [setup.py](https://github.com/spyder-ide/spyder-kernels/blob/master/setup.py#L42). It's very important to have it there too.